src/e2e-test/features/BQExecuter/BQExecute_WithMacro.feature

```diff
@@ -0,0 +1,29 @@
+@BQExecute
+Feature: BigQueryExecute - Verify successful data transfer to BQ table using BQExecute plugin with macro arguments
+
+  @BQ_SOURCE_TEST @BQ_SINK_TEST
+  Scenario:Validate successful records transfer Executes a SQL query in BigQuery Execute with macro arguments
```

Comment:
-> Verify data transfer of BigQueryExecute plugin with macro arguments

Reply:
@sawantpritam -done